### DISCUSSION TITLE
I need 15 stars for the Starstruck achievement

### DISCUSSION THREAD

**Discussion by guexs** [Schweinepriester/github-profile-achievements#190] [Q&A] [answered]
I need 15 stars for the Starstruck achievement, if you help me, I can help you if you need stars for the achievement.

Github Repo: https://github.com/Wannderlust/wannderlust

**Comment by Ysrbolles** [comment, marked as answer]
It has been starred. Make sure to star my repo https://github.com/Ysrbolles/Darkly-42Network

**Reply by 404jv** [reply]
I did it too. Please, leave a star in my repo https://github.com/404jv/dev-landia

**Reply by Ysrbolles** [reply]
Done

**Reply by guexs** [reply]
@Ysrbolles ready

**Reply by faintout** [reply]
> It has been starred. Make sure to star my repo https://github.com/Ysrbolles/Darkly-42Network

Done. Please leave a star here:
https://github.com/faintout/ThreeRoomDemo

**Reply by melvincwng** [reply]
⭐ - https://github.com/melvincwng/facebook-clone

**Reply by warriorwizard** [reply]
please leave a starr here too 🙏🙏 : https://github.com/warriorwizard/warriorwizard

**Comment by 404jv** [comment]
Done. Please, leave a star here https://github.com/404jv/dev-landia

**Reply by guexs** [reply]
Ready

**Reply by melvincwng** [reply]
https://github.com/melvincwng/facebook-clone

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by warriorwizard** [reply]
please leave a starr here too 🙏🙏 : https://github.com/warriorwizard/warriorwizard

**Reply by guexs** [reply]
Done, please leave a star: https://github.com/wannderlust/wannderlust

**Reply by warriorwizard** [reply]
Done,
Leave a star a star here  🙏🙏 : https://github.com/warriorwizard/warriorwizard
and mention ur repo link down below👇

**Comment by nel0x** [comment]
Did all above. Please star https://github.com/nel0x/arch-installer

**Reply by warriorwizard** [reply]
please leave a starr here too 🙏🙏 : https://github.com/warriorwizard/warriorwizard
i will surely star others,
btw thanks in advance

**Reply by nel0x** [reply]
@warriorwizard Done! Please also leave one here https://github.com/nel0x/arch-installer

**Reply by warriorwizard** [reply]
@nel0x thnx buddy i've done it too

**Comment by faintout** [comment]
Done. Please leave a star here:
https://github.com/faintout/ThreeRoomDemo

**Reply by warriorwizard** [reply]
please leave a starr here too 🙏🙏 : https://github.com/warriorwizard/warriorwizard
i will surely star others,
btw thanks in advance

**Comment by faintout** [comment]
> I need 15 stars for the Starstruck achievement, if you help me, I can help you if you need stars for the achievement.
> 
> Github Repo: https://github.com/Wannderlust/wannderlust

Done. Please leave a star here:
https://github.com/faintout/ThreeRoomDemo

**Reply by guexs** [reply]
Ready!

**Reply by warriorwizard** [reply]
please leave a starr here too 🙏🙏 : https://github.com/warriorwizard/warriorwizard
i will surely star others,
btw thanks in advance

**Comment by melvincwng** [comment]
For those who are interested to help ⭐:

1. https://github.com/melvincwng/facebook-clone
2. https://github.com/Your-First-Open-Source-Project/Your-First-Open-Source-Project

Thanks In advance, glad to see such a vibrant and inclusive community here on GitHub 😊

**Reply by faintout** [reply]
> 对于那些有兴趣提供帮助的人⭐：
> 
> 1. https://github.com/melvincwng/facebook-clone
> 2. https://github.com/Your-First-Open-Source-Project/Your-First-Open-Source-Project
> 
> 提前致谢，很高兴在 GitHub 上看到这样一个充满活力和包容性的社区😊

Done. Please leave a star here:
https://github.com/faintout/ThreeRoomDemo

**Reply by melvincwng** [reply]
@faintout , done ^^

**Reply by warriorwizard** [reply]
please leave a starr here too 🙏🙏 : https://github.com/warriorwizard/warriorwizard
i will surely star others,
btw thanks in advance

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by mezotv** [reply]
Done please leave one here https://github.com/mezotv/TodayIsThatDay

**Comment by guexs** [comment]
Done

**Comment by number0x0001** [comment]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by mezotv** [reply]
Done please leave one here https://github.com/mezotv/TodayIsThatDay

**Comment by warriorwizard** [comment]
please leave a star here too 🙏🙏 : https://github.com/warriorwizard/warriorwizard
btw thanks in advance

**Reply by mezotv** [reply]
Done please leave one here https://github.com/mezotv/TodayIsThatDay

**Comment by HellCatVN** [comment]
If you leave start here https://github.com/HellCatVN/HellCatVN 
I will star back your profile repo

**Reply by nel0x** [reply]
Both done! @HellCatVN Please leave one here https://github.com/nel0x/arch-installer

**Reply by mezotv** [reply]
Done please leave one here https://github.com/mezotv/TodayIsThatDay

**Comment by OmarHamdann** [comment]


**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by warriorwizard** [comment]
Done,
Leave a star a star here  🙏🙏 : https://github.com/warriorwizard/warriorwizard
and mention ur repo link down below👇

**Comment by erikn69** [comment]
Done,
Please help me here: https://github.com/erikn69/laravel-google-drive-ext-demo

**Reply by guexs** [reply]
Done

**Reply by mezotv** [reply]
Done please leave one here https://github.com/mezotv/TodayIsThatDay

**Comment by mezotv** [comment]
Done, please help me here https://github.com/mezotv/TodayIsThatDay

**Reply by guexs** [reply]
Done

**Reply by erikn69** [reply]
Done, please help me too
https://github.com/erikn69/laravel-google-drive-ext-demo
